src/LaunchDarkly.ServerSdk.Redis/Redis.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Net;
+using LaunchDarkly.Sdk.Server.Interfaces;
+
+namespace LaunchDarkly.Sdk.Server.Integrations
+{
+    /// <summary>
+    /// Integration between the LaunchDarkly SDK and Redis.
+    /// </summary>
+    public static class Redis
+    {
+        /// <summary>
+        /// The default location of the Redis server: <c>localhost:6379</c>
+        /// </summary>
+        public static readonly EndPoint DefaultRedisEndPoint = new DnsEndPoint("localhost", 6379);
+
+        /// <summary>
+        /// The default value for <see cref="RedisDataStoreBuilder.Prefix"/>.
+        /// </summary>
+        public static readonly string DefaultPrefix = "launchdarkly";
+
+        /// <summary>
+        /// The default value for <see cref="RedisDataStoreBuilder.ConnectTimeout(TimeSpan)"/>.
+        /// </summary>
+        public static readonly TimeSpan DefaultConnectTimeout = TimeSpan.FromSeconds(5);
+
+        /// <summary>
+        /// The default value for <see cref="RedisDataStoreBuilder.ResponseTimeout(TimeSpan)"/>.
+        /// </summary>
+        public static readonly TimeSpan DefaultResponseTimeout = TimeSpan.FromSeconds(3);
+
+        /// <summary>
+        /// The default value for <see cref="RedisDataStoreBuilder.OperationTimeout(TimeSpan)"/>.
+        /// </summary>
+        public static readonly TimeSpan DefaultOperationTimeout = TimeSpan.FromSeconds(3);
+
+        /// <summary>
+        /// Returns a builder object for creating a Redis-backed data store.
+        /// </summary>
+        /// <remarks>
+        /// This object can be modified with <see cref="RedisDataStoreBuilder"/> methods for any desired
+        /// custom Redis options. Then, pass it to <see cref="Components.PersistentDataStore(IPersistentDataStoreFactory)"/>
+        /// and set any desired caching options. Finally, pass the result to <see cref="ConfigurationBuilder.DataStore(IDataStoreFactory)"/>.
```

Comment:
The doc comments in this PR are all based on the wording in the equivalent Java library (https://github.com/launchdarkly/java-server-sdk-redis).